### PR TITLE
fix: remove 32 chars limit for identifiers 

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/AUT-1612

Remove 32 chars limit for item Identifier

### CODE REVIEW

File: views/js/qtiCreator/helper/itemLoader.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ define([
     const _generateIdentifier = function _generateIdentifier(uri) {
         const pos = uri.lastIndexOf('#');
         // identifier by default should be no more then 32
-        return uri.substr(pos + 1, qtiIdentifier.maxQtiIdLength);
+        return uri.substring(pos + 1, pos + 1 + qtiIdentifier.maxQtiIdLength);
```
Comment (pnal) [inline views/js/qtiCreator/helper/itemLoader.js:33]: where `maxQtiIdLength` is defined?

**Review by pnal — APPROVED** [review]
No FE unit tests
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Review by andreluizmachado — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [ ] Changelog is updated according to changes (if applicable)
- [x] Documentation is updated according to changes (if applicable)

**Review by bartlomiejmarszal — APPROVED** [review]

**Review by Silvia-loza — APPROVED** [review]
Review of FE side and tested in local. all working as expected
- [ ] New code is covered by tests (if applicable)
- [X] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [X]  Pull request's target is not` master`
- [X] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

**Pull request merged**